doc/tutorials/debug.rst

```diff
@@ -0,0 +1,371 @@
+.. _acrn-debug:
+
+ACRN Debugging Tools
+####################
+
+This document describes how to use ACRN tools to collect log
+and trace data for debugging.
+
+ACRN Console Command
```

Comment:
There is some interesting additional info in this doc but it really needs to be combined/merged with https://projectacrn.github.io/latest/user-guides/acrn-shell.html. We do not want to keep multiple copies of the same info as it will quickly go out of sync.